src/app/offer-form/offer-form.component.scss

```diff
@@ -0,0 +1,21 @@
+::ng-deep {
```

Comment:
Avoid using ::ng-deep.

Reply:
But I exactly do want all the labels and form-field inserted via `ng-content` to "inherit" these styles. Any alternatives?

Reply:
Make all this styles global but wrapped in element selector:
1. Create file app-offer-form.scss in assets\css\components
2. In app-offer-form.scss add:
app-offer-form {
  // only your ng-deep css, rest should be in component style file
}
3. Import file in styles.scss

This is how I aproach the topic because angular plan to drop support for ::ng-deep.